Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_MounriGallatross.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+		<li>Alpha Animals</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/ThingDef[defName="AA_GallatrossMoribund"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Quadruped</bodyShape>
+					</li>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>/Defs/ThingDef[defName="AA_GallatrossMoribund"]/statBases</xpath>
+				<value>
+					<ArmorRating_Blunt>26</ArmorRating_Blunt>
+					<ArmorRating_Sharp>13</ArmorRating_Sharp>
+					<MeleeDodgeChance>0.02</MeleeDodgeChance>
+					<MeleeCritChance>1.75</MeleeCritChance>
+					<MeleeParryChance>0.3</MeleeParryChance>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+		                 <xpath>/Defs/ThingDef[defName="AA_GallatrossMoribund"]/race/baseHealthScale</xpath>
+		                 <value>
+			               <baseHealthScale>8</baseHealthScale>
+		                 </value>
+
+	                </li>
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="AA_GallatrossMoribund"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>horn</label>
+							<capacities>
+								<li>Blunt</li>
+							</capacities>
+							<power>86</power>
+							<cooldownTime>2.4</cooldownTime>
+							<linkedBodyPartsGroup>HornAttackTool</linkedBodyPartsGroup>
+							<armorPenetrationBlunt>70.226</armorPenetrationBlunt>	
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<capacities>
+								<li>Bite</li>
+							</capacities>
+							<power>60</power>
+							<cooldownTime>2.0</cooldownTime>
+							<linkedBodyPartsGroup>Teeth</linkedBodyPartsGroup>
+							<armorPenetrationSharp>11</armorPenetrationSharp>
+							<armorPenetrationBlunt>22</armorPenetrationBlunt>	
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>head</label>
+							<capacities>
+								<li>Blunt</li>
+							</capacities>
+							<power>46</power>
+							<cooldownTime>2</cooldownTime>
+							<linkedBodyPartsGroup>HeadAttackTool</linkedBodyPartsGroup>
+							<armorPenetrationBlunt>31</armorPenetrationBlunt>	
+							<chanceFactor>0.1</chanceFactor>
+						</li>
+					</tools>
+				</value>
+			</li>
+			
+			<!-- Add Low Bleed Hediff -->
+
+			<li Class="PatchOperationAdd"> 
+			<xpath>/Defs</xpath>
+				<value>
+  <DamageDef ParentName="StunBase">
```

Comment:
What's the purpose of this? It appears to be a duplicate of the vanilla DamageDef?

Reply:
Yes it's a duplicate I think, I copypasted the original gallatross patch, doesn't result in any errors tho.

Reply:
In theory, it shouldn't because it's just overwriting the old thingDef, but if it's not necessary, it really should be deleted--needlessly overwriting stuff can complicated troubleshooting in the future when things don't behave how they're supposed to.